tests/index.js

```diff
@@ -45,6 +54,14 @@ describe("index.js", function () {
                         done();
                     });
                 });
+
+                it("should have the right doc contents", function () {
+                    const path = `./docs/rules/${fileName}.md`;
+                    const fileContents = fs.readFileSync(path, "utf8");
+
+                    const titleRegexp = new RegExp(`# .+ \\(${fileName}\\)`);
```

Comment:
I think this regular expression could be strengthened to make sure this is really an `<h1>` title, by matching start-of-line. What do you think?

```suggestion
                    const titleRegexp = new RegExp(`^# .+ \\(${fileName}\\)$`, "m");
```

Reply:
Thanks! I was trying to get that to work.